net/interfaces/interfaces.go

```diff
@@ -594,10 +629,7 @@ func anyInterestingIP(pfxs []netaddr.IPPrefix) bool {
 // should log in interfaces.State logging. We don't need to show
 // localhost or link-local addresses.
 func isInterestingIP(ip netaddr.IP) bool {
-	if ip.IsLoopback() || ip.IsLinkLocalUnicast() {
```

Comment:
Arbitrary churn but sure :)

Reply:
I had to read it twice to make sure I understood it. (If this, then false?) This helped me, so figured it'd help the next person.